src/sentry/static/sentry/app/views/organizationEventsV2/sortLink.jsx

```diff
@@ -9,21 +9,26 @@ class SortLink extends React.Component {
   static propTypes = {
     title: PropTypes.string.isRequired,
     sortKey: PropTypes.string.isRequired,
+    implicitSort: PropTypes.string,
```

Comment:
What does `implicitSort` mean? Would defaultSort be a better name? Should it be a required value since every view has a default sort?

Reply:
I used 'implicit' as the sorting value is implied by the default view. I'm good to use `defaultSort` if that is simpler to understand.